pkg/common/resources/resources.go

```diff
@@ -792,7 +792,7 @@ func CalculateAbsUsedCapacity(capacity, used *Resource) *Resource {
 				absResValue = math.MinInt64
 			}
 		} else {
-			log.Logger().Warn("Cannot calculate absolute usage for resource because of missing usage information",
+			log.Logger().Info("Cannot calculate absolute usage for resource because of missing usage information",
```

Comment:
Today, we are reporting back all resource types, so we constantly see these lines because resource types "pods", "vcore", "ephemeral-storage", "hugepages-1Gi" etc are not tracked in the scheduler. So setting at INFO level won't seem to be enough for this. I'd suggest removing this line.
Or we will need some mechanism to do white-listed resource names, that will introduce bigger changes for which I don't think it is necessary for now.